seed.js

```diff
@@ -1,42 +1,167 @@
 // Load Environment
-var dotenv = require('dotenv');
+const dotenv = require('dotenv');
 dotenv.load();
 process.env.MONGO_DATABASE = 'mongodb://localhost:27017/connect';
 process.env.OUTPUT_LOG = './log.txt';
 
 // create a password
-var crypto = require('crypto');
-var password = ''
-crypto.randomBytes(16).forEach(value => {
-  password += (value % 16).toString(16);
-})
-
-var app = require('express')();
+const crypto = require('crypto');
+const app = require('express')();
 
 // stub out the schema
 require('./app/helpers/database')(app);
 
 // create an admin user
-var access = require('./app/helpers/access');
-var User = require('./app/models/user');
+const User = require('./app/models/user');
+const Game = require('./app/models/game');
+const Group = require('./app/models/group');
+const uuid = require('uuid/v1');
+const Release = require('./app/models/release');
+
+function makeRandomString(length){
+  let random = '';
+  crypto.randomBytes(length).forEach(value => {
+    random += (value % length).toString(length);
+  });
+  return random;
+}
 
-console.log('Creating admin user...');
-User.createUser(
-  {
+async function makeDummyData(){
+  let admin = await makeAdmin();
+  // let game = await makeGame();
```

Comment:
Looks like this is commented out. Any particular reason?

Reply:
Its a mistake from my local testing, let me clean this up real uick